src/components/SideColumns/SideColumns.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "styled-components";
+import { Stack, H2, useDeskproAppTheme } from "@deskpro/app-sdk";
+
+import { GreyTitle } from "../../styles";
+import { ReactElement } from "react";
+
+const Divider = styled.div`
+  display: inline-block;
+  width: 1px;
+  height: 2.5em;
```

Comment:
what if the columns height is greater than 2 lines? or if the columns will be different heights?

Reply:
gets resized

Reply:
How if here hardcoded height?